client/app/bundles/shared/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,67 @@
+@import "../base.scss";
+
+body {
+  font-family: $font-family;
+}
+
+.fill select{
+  @include selectDropdown(
+    $background: #6D0839,
+    $width: 143px,
```

Comment:
Let's make the width flexible here so that we're not always bound to a fixed width!